src/main.rs

```diff
@@ -96,14 +88,12 @@ fn notmain() -> Result<i32> {
         let mut new_linker_script = File::create(tempdir.join(ram_linker_script.file_name()))?;
 
         for (index, line) in original_linker_script.lines().enumerate() {
-            if index == ram_entry.line {
-                writeln!(
+            match index == ram_entry.line {
```

Comment:
Why do you `match` on a bool, instead of using `if`? I never saw this syntax before associated with `match`.

Reply:
It does not make much difference if you use `match` or `if-else`. I just find that it often saves quite a few lines of code when using `match`.

Reply:
I see. I think the intention is less evident.

Reply:
Then I will change it back if you prefer that.